### PR TITLE
SLI-1436 Analysis in CLion on Windows does not work

### PR DESCRIPTION
**Pull request by nquinquenel**

### CODE REVIEW

File: src/main/java/org/sonarlint/intellij/util/VirtualFileUtils.kt [hunk 1]
```diff
@@ -28,7 +28,7 @@ import java.net.URISyntaxException
 object VirtualFileUtils {
     fun toURI(file: VirtualFile): URI? {
         return try {
-            URI(file.url.replace(" ", "%20"))
+            URI("${file.fileSystem.protocol}:///${file.path}".replace(" ", "%20"))
```
Comment (damien-urruty-sonarsource) [inline src/main/java/org/sonarlint/intellij/util/VirtualFileUtils.kt:31]: My concern is that this won't work with remote files. But maybe we already don't support this case with SLI?
Also I think you could at least print a log in the catch below

**Review by damien-urruty-sonarsource — APPROVED** [review]
I think this works for most cases, except remote FS. Not sure we need to support this case

File: src/main/java/org/sonarlint/intellij/util/VirtualFileUtils.kt (hunk 1 above)
Comment (damien-urruty-sonarsource) [reply src/main/java/org/sonarlint/intellij/util/VirtualFileUtils.kt:31]: You could maybe check if the `file.isInLocalFileSystem()`? If it's not you will try to build a local URI that maybe does not exist

Comment (nquinquenel) [reply src/main/java/org/sonarlint/intellij/util/VirtualFileUtils.kt]: We were effectively only computing URI for file in local file system before SLOOP

**Pull request merged**